Message interviews:21 -  Read
From:     Paul Calder <calder@aramis.stanford.edu>
To:       <interviews@interviews.stanford.edu>
Reply-To: <calder@lurch.stanford.edu>
Subject:  fix for TextDisplay bug.

TextDisplay was not resetting the text style of its painter.  If the
painter was shared (a common case), other text subseqently drawn with
the painter would appear with incorrect styling.

This patch is available for ftp as fix14
---
Apply with ``cd iv; patch -p -N < diffs'' to patch files:

        src/libInterViews/textdisplay.c

*** /tmp/,RCSt1a20504                     Tue Jul 11 11:52:59 1989
--- src/libInterViews/textdisplay.c       Tue Jul 11 11:52:32 1989
***************
*** 736,741 ****
--- 736,742 ----
                      }
                  }
              }
+             display->painter->SetStyle(Plain);
              if (last > lastchar) {
                  Coord r, y;
                  display->painter->GetPosition(r, y);
